packages/core/src/helpers/getTextContentFromNodes.ts

```diff
@@ -0,0 +1,17 @@
+import { ResolvedPos } from 'prosemirror-model'
+
+export const getTextContentFromNodes = ($from: ResolvedPos<any>, maxMatch = 500) => {
```

Comment:
Maybe export this in the `index.ts` file like all other helpers?

Reply:
thanks, added